school/migrations/0001_initial.py

```diff
@@ -0,0 +1,123 @@
+# Generated by Django 3.1.7 on 2021-03-29 22:38
+
+from django.conf import settings
+import django.contrib.auth.models
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
```

Comment:
Given that this file is auto-generated, why include it in the PR?  I am not sure what the benefit of tracking this in the repo would be.